_rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md

```diff
@@ -0,0 +1,100 @@
+---
+name: ARIA allowed attribute
+description: | 
+  This rule checks that `aria-*` attributes are allowed for the element they are specified on.
+
+success_criterion:
+- 4.1.1 # Parsing (A)
+- 4.1.2 # Name, Role, Value (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+Any attribute that is placed on an HTML or SVG element where the attribute name starts with `aria-`.
+
+### Expectation
+
+The `aria-*` attribute is either listed under [Supported States and Properties](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) in [WAI-ARIA](https://www.w3.org/TR/wai-aria) for the [semantic role](#semantic-role) of the element that the attribute is specified on, or it is found on the list of [Global States and Properties](https://www.w3.org/TR/wai-aria-1.1/#global_states).
```

Comment:
Minor suggestions:
- "The `aria-*` attribute is" -> "The target attribute is", to avoid rephrasing the applicability
- Say "WAI-ARIA supported states and properties", to drop the second link to WAI-ARIA
- Add "WAI-ARIA 1.1" to the phrase "global states and properties" to identify the source